### DISCUSSION TITLE
How to convert the doris rpc function bytes to BigDecimal

### DISCUSSION THREAD

**Discussion by foundation1** [apache/doris#31246] [A - General / Q&A]
字段类型为 DECIMAL(16,3)  通过自定义函数 接收的值是一个bytes
我需要怎么转换为一个 bigdecimal啊
![image](https://github.com/apache/doris/assets/28500506/93d54a14-ecd6-4dca-be42-05517f9e593b)
![image](https://github.com/apache/doris/assets/28500506/da9d6393-db4a-4f79-ad31-1080a90da057)

**Comment by yagagagaga** [comment]
It's suggest to using [Java UDF](https://doris.apache.org/zh-CN/docs/ecosystem/udf/java-user-defined-function)
